packages/create-snap/package.json

```diff
@@ -8,15 +8,30 @@
   },
   "license": "ISC",
   "sideEffects": false,
-  "bin": "./dist/main.js",
+  "exports": {
+    ".": {
+      "import": "./dist/esm/index.js",
+      "require": "./dist/cjs/index.js",
+      "types": "./dist/types/index.d.ts"
+    }
+  },
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
+  "types": "dist/types/index.d.ts",
+  "bin": "dist/cjs/main.js",
```

Comment:
Does this need to be `./` prefixed? 🤔

Reply:
I don't think it's necessary, but I did it anyway for consistency.